CI failure seems to be another instance of https://github.com/bitcoin/bitcoin/issues/30922

```
test  2024-09-22T19:16:03.915000Z TestFramework (ERROR): Assertion failed 
                                   Traceback (most recent call last):
                                     File "/Users/runner/work/bitcoin/bitcoin/test/functional/test_framework/test_framework.py", line 132, in main
                                       self.run_test()
                                     File "/Users/runner/work/bitcoin/bitcoin/ci/scratch/build-aarch64-apple-darwin23.6.0/test/functional/p2p_1p1c_network.py", line 137, in run_test
                                       self.raise_network_minfee()
                                     File "/Users/runner/work/bitcoin/bitcoin/ci/scratch/build-aarch64-apple-darwin23.6.0/test/functional/p2p_1p1c_network.py", line 58, in raise_network_minfee
                                       assert_greater_than(node.getmempoolinfo()['mempoolminfee'], FEERATE_1SAT_VB)
                                     File "/Users/runner/work/bitcoin/bitcoin/test/functional/test_framework/util.py", line 82, in assert_greater_than
                                       raise AssertionError("%s <= %s" % (str(thing1), str(thing2)))
                                   AssertionError: 0.00001000 <= 0.00001000
```

--- a/doc/zmq.md
+++ b/doc/zmq.md
@@ -46,11 +46,10 @@ operation.
 
 ## Enabling
 
-By default, the ZeroMQ feature is automatically compiled in if the
-necessary prerequisites are found.  To disable, use -DWITH_ZMQ=OFF
-during the *configure* step of building bitcoind:
+By default, the ZeroMQ feature is not automatically compiled.
+To enable, use `-DWITH_ZMQ=ON` when configuring the build system:
 
-    $ cmake -B build -DWITH_ZMQ=OFF (other options)
+    $ cmake -B build -DWITH_ZMQ=ON
 
 To actually enable operation, one must set the appropriate options on
 the command line or in the configuration file.

